docroot/themes/humsci/humsci_basic/src/scss/components/_masthead.scss

```diff
@@ -4,7 +4,21 @@
   border-bottom: $hb-thin-border;
   @include hb-global-color('border-color', 'gray');
 
+  .su-site-header-nav-container {
+    display: grid;
```

Comment:
:clap: Nice! This looks like it has the same affect I proposed in the ticket. Good call refactoring this a bit more.